### ISSUE TITLE
Google2FA access level must be public

### ISSUE THREAD

**Issue by Cl1608Ho** [monicahq/monica#3497]
I'm getting this error when loading the page on my own installation, i've enabled webauthn yesterday (but just webauthn, no TOTP 2fa

```
Access level to PragmaRX\Google2FALaravel\Support\Config::config() must be public (as in class PragmaRX\Google2FALaravel\Support\Auth)
```

**Which version are you using:**
 - A server you maintain yourself v2.16.0

**Additional context**
Just appeared today

**Comment by Cl1608Ho** [comment]
Any help would be greatly appreciated

**Comment by asbiin** [comment]
@Cl1608Ho I will look at this next week. In the meantime, you can remove entries in the `webauthn_keys` table to have access to your account.

**Comment by Cl1608Ho** [comment]
I did remove the entry, the error persisted anyway.

**Comment by asbiin** [comment]
This is the same issue as in https://github.com/antonioribeiro/google2fa-laravel/issues/96
What is your php version?

**Comment by Cl1608Ho** [comment]
This indeed solved the issue. I tried multiple php versions, currently running `7.4.2`. I hope a quick dependency update will resolve this.

**Comment by Cl1608Ho** [comment]
There has been an upstream patch solving antonioribeiro/google2fa-laravel#96

Would be good if you included the patch and prepared for a monica patch release.

**Comment by Cl1608Ho** [comment]
Bump

**Comment by asbiin** [comment]
already fixed by #3796
